packages/discovery-provider/ddl/migrations/0074_usdc_purchase_splits_user_ids.sql

```diff
@@ -0,0 +1,175 @@
+BEGIN TRANSACTION;
```

Comment:
I believe my understanding is right here, but just confirming -- 

tracks inherit from their parent album at upload time, so we will catch both by this migration. no way to get those out of sync here.

Reply:
both are updated in the same transaction anyway, so there's no sync issue

Reply:
I stand corrected - fixed